python/flink_agents/plan/tests/test_action.py

```diff
@@ -58,14 +60,22 @@ def action() -> Action:  # noqa: D103
         name="legal",
         exec=func,
         listen_event_types=[f"{InputEvent.__module__}.{InputEvent.__qualname__}"],
+        params={
```

Comment:
`Action` is not a public api. The question is, do we want to expose the ability of adding parameters to users. If true, how can user use this feature via public apis? If not, we should not expose this via runner context, which is also public api.